test/test_mps.py

```diff
@@ -6405,543 +6406,3244 @@ class TestConsistency(TestCase):
-                    'torch.uint8'],
-        'stack': ['torch.float16',
-                  'torch.float32',
-                  'torch.int16',
-                  'torch.int32',
-                  'torch.int64',
-                  'torch.uint8'],
-        'sub': ['torch.float32',
-                'torch.int16',
-                'torch.int32',
-                'torch.int64'],
-        'sum_to_size': ['torch.bool',
-                        'torch.float16',
-                        'torch.float32',
-                        'torch.int16',
-                        'torch.int32',
-                        'torch.int64',
-                        'torch.uint8'],
-        'svd': ['torch.float32'],
-        't': ['torch.bool',
-              'torch.float16',
-              'torch.float32',
-              'torch.int16',
-              'torch.int32',
-              'torch.int64',
-              'torch.uint8'],
-        'tanh': ['torch.float32'],
-        'tensordot': ['torch.float32'],
-        'topk': ['torch.float32'],
-        'tril': ['torch.bool',
-                 'torch.float16',
-                 'torch.float32',
-                 'torch.int16',
-                 'torch.int32',
-                 'torch.int64',
-                 'torch.uint8'],
-        'triu': ['torch.bool',
-                 'torch.float16',
-                 'torch.float32',
-                 'torch.int16',
-                 'torch.int32',
-                 'torch.int64',
-                 'torch.uint8'],
-        'true_divide': ['torch.float32'],
-        'trunc': ['torch.float32'],
-        'unsqueeze': ['torch.bool',
-                      'torch.float16',
-                      'torch.float32',
-                      'torch.int16',
-                      'torch.int32',
-                      'torch.int64',
-                      'torch.uint8'],
-        'view': ['torch.bool',
-                 'torch.float16',
-                 'torch.float32',
-                 'torch.int16',
-                 'torch.int32',
-                 'torch.int64',
-                 'torch.uint8'],
-        'view_as': ['torch.bool',
-                    'torch.float16',
-                    'torch.float32',
-                    'torch.int16',
-                    'torch.int32',
-                    'torch.int64',
-                    'torch.uint8'],
-        'vsplit': ['torch.bool',
-                   'torch.float16',
-                   'torch.float32',
-                   'torch.int16',
-                   'torch.int32',
-                   'torch.int64',
-                   'torch.uint8'],
-        'vstack': ['torch.float16',
-                   'torch.float32',
-                   'torch.int16',
-                   'torch.int32',
-                   'torch.int64'],
-        'zero_': ['torch.float16',
-                  'torch.float32',
-                  'torch.int16',
-                  'torch.int32',
-                  'torch.int64',
-                  'torch.uint8']}
+        "__getitem__": [
```

Comment:
I feel like these either need to become blocklist (as there are more enabled than disabled) or we should split it in a different file to keep this one at a reasonable size.

Reply:
I've updated to use abbreviations for the dtypes (aka f16 instead of torch.float16) to reduce the size, perhaps it is more reasonable now?